components/team_settings/team_settings.tsx

```diff
@@ -1,13 +1,28 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import PropTypes from 'prop-types';
 import React from 'react';
 
 import GeneralTab from 'components/team_general_tab';
 import ImportTab from 'components/team_import_tab.jsx';
 
-const TeamSettings = ({activeTab, activeSection, updateSection, closeModal, collapseModal, team}) => {
+type Props = {
+    activeTab: string;
+    activeSection: string;
+    updateSection: () => void;
+    closeModal: () => void;
+    collapseModal: () => void;
+    team?: Record<string, unknown>;
+};
+
+const TeamSettings = ({
+    activeTab = '',
+    activeSection = '',
+    updateSection,
+    closeModal,
+    collapseModal,
+    team,
+}: Props): JSX.Element | null | undefined => {
```

Comment:
Wonder in which cases will this be undefined?

Reply:
Mistakenly added. Removed in my recent commits.